code/game/gamemodes/clock_cult/clock_structures/geis_binding.dm

```diff
@@ -106,19 +149,15 @@
 	animate(T2, pixel_y = pixel_y - 9, alpha = 0, time = 8, easing = EASE_IN)
 
 /obj/structure/destructible/clockwork/geis_binding/user_unbuckle_mob(mob/living/buckled_mob, mob/user)
-	if(buckled_mob == user)
-		if(!resisting && can_resist)
-			resisting = TRUE
-			user.visible_message("<span class='warning'>[user] starts struggling against [src]...</span>", "<span class='userdanger'>You start breaking out of [src]...</span>")
-			while(do_after(user, 10, target = src) && resisting && obj_integrity)
-				if(obj_integrity - 5 <= 0)
-					user.visible_message("<span class='warning'>[user] breaks [src]!</span>", "<span class='userdanger'>You break [src]!</span>")
-					take_damage(5)
-					return user
-				take_damage(5)
-			resisting = FALSE
-	else
-		return ..()
+	if(resisting)
+		to_chat(user, "<span class='warning'>You're already trying to break free!</span>")
+		return
+	if(is_servant_of_ratvar(user))
+		take_damage(obj_integrity) //freedom!
+		return
+	resisting = user
+	user.visible_message("<span class='warning'>[user] starts struggling against [src]...</span>", "<span class='userdanger'>You start breaking out of [src]...</span>")
```

Comment:
My main concern is that people won't understand that they're being broken out and start saying "hey this is bullshit I can't break free", so I'm kind of thinking that it might be worth keeping in for that